openapi3filter/validate_request.go

```diff
@@ -200,14 +210,18 @@ func ValidateSecurityRequirements(c context.Context, input *RequestValidationInp
 		return nil
 	}
 
-	doneChan := make(chan bool, len(srs))
+	var wg sync.WaitGroup
 	errs := make([]error, len(srs))
 
-	// For each alternative
+	// For each alternative security requirement
 	for i, securityRequirement := range srs {
 		// Capture index from iteration variable
 		currentIndex := i
 		currentSecurityRequirement := securityRequirement
+
+		// Add a work item
+		wg.Add(1)
```

Comment:
This can be replaced by `wg.Add(len(srs))` before the loop. It supposedly saves a bit of time.